src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  GraphQLInterfaceType,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLUnionType,
+} from "graphql"
+import { InternalIDFields, NodeInterface } from "../object_identification"
+import { HomeViewComponent } from "./HomeViewComponent"
+
+// section interface
+
+import { ResolverContext } from "types/graphql"
+const GenericHomeViewSectionInterface = new GraphQLInterfaceType({
+  name: "GenericHomeViewSection",
+  description: "Abstract interface shared by every kind of home view section",
+  fields: {
+    ...InternalIDFields,
+    key: {
+      type: GraphQLNonNull(GraphQLString),
+      description: "A stable identifier for this section",
+    },
+    title: {
+      type: GraphQLNonNull(GraphQLString),
+      description: "A display title for this section",
+    },
+    component: {
```

Comment:
maybe something more generic like `section` would be better? (Thinking its best to keep our data layer as generic as possible, vs encoding react idioms)

Reply:
+1, it's unfortunate that React has monopolized the word "component" in my mind, but I totally assumed that this pointed to a specific type of React component that should be rendered by the client.

Reply:
+1, it's unfortunate that React has monopolized the word "component" in my mind, but I totally assumed that this pointed to a specific type of React component that should be rendered by the client.